### PR TITLE
Minor adjustments for stage select

### PR DESCRIPTION
**Pull request by JaceCear**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -84,7 +84,7 @@ TOOLS = $(foreach tool,$(TOOLBASE),tools/$(tool)/$(tool)$(EXE))
 # -I sets an include path
 # -D defines a symbol
 CPPFLAGS ?= -iquote include -D $(GAME_REGION)
-CC1FLAGS ?= -Wimplicit -Wparentheses -Werror -Wno-parentheses-equality
```
Comment (freshollie) [inline Makefile:87]: Yeah fare enough removing this it's probably not needed

File: src/sakit/unused_level_select.c [hunk 2]
```diff
@@ -14,8 +14,6 @@
 
 #include "constants/songs.h"
 
-const char ALIGNED(4) gUnknown_080D5128[8] = "STAGE";
```
Comment (freshollie) [inline src/sakit/unused_level_select.c:17]: Where is this now?

File: src/data/tilemap_debug_text.c [hunk 3]
```diff
@@ -0,0 +1,4 @@
+#include "data/tileset_language.h"
+
+// Tiles used for debug strings, like the unused (in the retail version) Stage Select.
+const u8 Tileset_Language[0] = {};
```
Comment (freshollie) [inline src/data/tilemap_debug_text.c:4]: Do you not wanna put these tiles into the repo?

File: src/sakit/unused_level_select.c (hunk 2 above)
Comment (JaceCear) [reply src/sakit/unused_level_select.c:17]: In the function call! ^^

**Review by freshollie — APPROVED** [review]

File: src/sakit/unused_level_select.c (hunk 2 above)
Comment (freshollie) [reply src/sakit/unused_level_select.c:17]: Wow, very good! I'm sure there are other places where this could be done

**Pull request merged**